### DISCUSSION TITLE
Prompt generation does not start

### DISCUSSION THREAD

**Discussion by MisterMagoo1689** [imrayya/stable-diffusion-webui-Prompt_Generator#18] [Q&A]
Cannot understand why. I've used a simple prompt like "portrait of a girl with blond hair, smiling, dressed in a nineteenth-century dress, with a steampunk city in the background", using the Stable Diffusion 2.1 model. I do not have any other extension loaded. 

I click on "Generate", nothing happens. The DOS windows behind is also inactive, no processes are starting. 

python: 3.10.6  •  torch: 1.13.1+cu117  •  xformers: 0.0.16rc425  •  gradio: 3.23.0  •  commit: [22bcc7be](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/22bcc7be428c94e9408f589966c2040187245d81)  •  checkpoint: [cc6cb27103](https://google.com/search?q=cc6cb27103417325ff94f52b7a5d2dde45a7515b25c255d8e396c90014281516)

Do you have any idea? I've tried to delete & reinstall the extension already... no results

Thanks!

**Comment by JustLynamo** [comment]
Same issue here. Hoping for a fix.

**Comment by imrayya** [comment]
May I ask a few quick questions?

1. Is this your first time generating a longer prompt?
2. Within the directory `*username*/.cache/huggingface/hub/models`, does the folder contain at least one of the following: `models--gpt2` or `models--distilgpt2`?
3. Upon clicking the `Generate` button, do you encounter either of the following messages: `Exception encountered while attempting to install tokenizer` or `Generate new prompt from...`?

If there is no response to question # 3, it could indicate that the model is still initializing or being downloaded in the background. And the solution is to just wait until that finishes initializing or to wait for it to download

If you see `Generate new prompt from:...` that makes it more complicated to find out what the problem is

**Reply by MisterMagoo1689** [reply]
Thanks for your message: without me doing anything, this morning it is working perfectly! 
The only thing that changed between yesterday and today is that... I've restarted the PC. Unbevlievable... but problem "solved" :)
Answering to your questions:
1) yes, it was the first time I tried, I've just installed your extension
2) yes, yesterrday I checked already, and "distilgpt2" was there
3) upon clicking the "Genrate" button, NOTHING happened. No messages, no changes in the UI whatsoever.

Thanks! Great extension, BTW :)

Bye!
A.-

**Reply by imrayya** [reply]
I'm glad I could help even though I haven't really done anything. Thanks for the praise
